.github/workflows/black.yml

```diff
@@ -0,0 +1,15 @@
+name: Check code format
+
+on: [pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v3
+      - name: Black format check
```

Comment:
you're missing the step to configure the python version

Reply:
Done!